hardware_interface/src/component_parser.cpp

```diff
@@ -68,7 +69,8 @@ std::string get_text_for_element(
   const auto get_text_output = element_it->GetText();
   if (!get_text_output)
   {
-    throw std::runtime_error("text not specified in the " + tag_name + " tag");
+    std::cerr << "text not specified in the " << tag_name << " tag" << std::endl;
```

Comment:
I did not open a PR in the first place, since the desired behavior wasn't obvious to me. Is this to be considered a non-expected situation (as printing to `std::err` still suggests)?